src/firmware/10-snowbots.rules

```diff
@@ -0,0 +1,5 @@
+# Snowbots udev Rules 
```

Comment:
Can delete, has been moved to `setup_scripts`.

Reply:
done